control-plane/dist/update/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
Please remove this since we don't allow updates of jobs in the control plane.

Reply:
Done